_data/simple-icons.json

```diff
@@ -7708,6 +7708,11 @@
             "source": "https://www.jetbrains.com/company/brand/logos/",
             "guidelines": "https://www.jetbrains.com/company/brand/"
         },
+        {
+            "title": "Rimac",
```

Comment:
While I agree "Rimac" is the better option, I wonder whether this should be "Rimac automobili" :thinking: What do you think?

Reply:
Hi. Yes, in my opinion, Rimac automobili should be appropriate since it is the full brand name. As above, will amend this.